### ISSUE TITLE
Unnecessary tight version constraint limits FastAPI versions

### ISSUE THREAD

**Issue by graipher** [trallnag/prometheus-fastapi-instrumentator#136]
Due to this line in the [`pyproject.toml`](https://github.com/trallnag/prometheus-fastapi-instrumentator/blob/1ef3f430993a1aa1bbd271fe4e56a45a6823cde1/pyproject.toml#L17) file:

```
fastapi = "^0.38.1"
```

FastAPI versions newer than 0.38 cannot be used with this (current version of FastAPI is [0.75.2](https://pypi.org/project/fastapi/)). When explicitly requesting a higher version the version solving fails (using poetry):

```
$ poetry update
Updating dependencies
Resolving dependencies... (0.0s)

  SolverProblemError

  Because prometheus-fastapi-instrumentator (5.8.0) depends on fastapi (>=0.38.1,<0.39.0)
   and no versions of prometheus-fastapi-instrumentator match >5.8.0,<6.0.0, prometheus-fastapi-instrumentator (>=5.8.0,<6.0.0) requires fastapi (>=0.38.1,<0.39.0).
  So, because my-repo depends on both fastapi (^0.75.0) and prometheus-fastapi-instrumentator (^5.8.0), version solving failed.
```

One solution would be relaxing the requirements:

```
fastapi = "^0.38"
```
or

```
fastapi = ">=0.38.1, <1.0.0"
```

**Comment by alexted** [comment]
yeah, i have the same problem

**Comment by graipher** [comment]
I hot-fixed it by requiring `"^5.7.1` for prometheus-fastapi-instrumentator in my repository for now. But an actual fix would be nice of course.

**Comment by trallnag** [comment]
Thanks for raising the issue
